### DISCUSSION TITLE
Is there a config option to ignore `OnlyShowIn`?

### DISCUSSION THREAD

**Discussion by 0xf0xx0** [davatorium/rofi#1689] [Q&A]
I didn't see anything in the docs about it, so I'll ask here. Some of my apps, like my settings, have `OnlyShowIn` set. But since i use i3, rofi won't display any of them. Is there a config option or something i can change to fix that? [This other discussion](https://github.com/davatorium/rofi/discussions/1670) is similar but not exactly what i want.

**Comment by DaveDavenport** [comment]
No, it follows the standard.
It uses `XDG_CURRENT_DESKTOP` environment to determine if it should show.

**Reply by 0xf0xx0** [reply]
Well yeah, but i need a way to bypass that. i use i3, so "OnlyShowIn" for those apps never lists my WM. Even setting `XDG_CURRENT_DESKTOP` to something common like `Gnome` or `Cinnamon` has no effect.

**Reply by DaveDavenport** [reply]
Did you set the value right? It is (for me) `GNOME` not `Gnome`

```
XDG_CURRENT_DESKTOP=GNOME rofi -show drun
```
works fine for me to show desktop files that have 
```
OnlyShowIn=GNOME;
```
set.

Also unsetting XDG_CURRENT_DESKTOP should work as OnlyShowIn and NowShowIn are then ignored.

**Reply by DaveDavenport** [reply]
Unless it is in violation with the standard, this option won't be added.
There are multiple methods to fix this within the standard.
